docs/src/prototypes/Prototypes.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import { Box, Header, Segment } from '@stardust-ui/react'
+
+interface PrototypeSectionProps {
+  title?: string
+}
+
+interface ComponentPrototypeProps extends PrototypeSectionProps {
+  description?: string
+}
+
+export const PrototypeSection: React.SFC<ComponentPrototypeProps> = props => (
```

Comment:
```suggestion
export const PrototypeSection: React.FC<ComponentPrototypeProps> = props => (
```

SFC was deprecated :)

Reply:
done, thanks